src/js/pages/Hukamnama/index.js

```diff
@@ -2,11 +2,21 @@
 import React from 'react';
 import { buildApiUrl } from '@sttm/banidb';
 import PageLoader from '../PageLoader';
+import PropTypes from 'prop-types';
 import Layout, { Stub } from './Layout';
 
 export default class Hukamnama extends React.PureComponent {
+  static propTypes = {
+    location: PropTypes.shape({
+      hash: PropTypes.string,
+      pathname: PropTypes.string,
+    }),
+    date: PropTypes.string,
+    match: PropTypes.object.isRequired,
+  };
   render() {
-    const url = buildApiUrl({ hukam: true, API_URL });
+    let url = buildApiUrl({ hukam: true, API_URL });
+    url = url.replace('today', this.props.date);
 
```

Comment:
We should update the buildApiUrl function to accept date instead of boolean. Hacks bite in future.

Reply:
Opened up a pull request: https://github.com/KhalisFoundation/banidb/pull/4

Reply:
Once it's accepted and pushed, will do the change in sttm-web code.

Reply:
Updated

Reply:
No need of `let` anymore.